wandb/sdk/service/service_base.py

```diff
@@ -22,8 +21,6 @@ def _pbmap_apply_dict(
 ) -> None:
 
     for k, v in d.items():
-        if isinstance(v, datetime.datetime):
```

Comment:
just curious, why were we skipping it before?

Reply:
not sure. @raubitsj ?